README.md

```diff
@@ -1,18 +1,22 @@
 ﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | @nasaku898
-* Émilie Martin | @emilie-martin
-* Daniela Venuta | @daniela-venuta
-* Minh-Tam Do | @mint47
-* Luc Nguyen | @1999lucnguyen
-* Karin Kazarian | @Karin-kazarian
+Simon Lim (tech lead) | @nasaku898
+:--|:--
+Émilie Martin         | @emilie-martin  
+Daniela Venuta        | @daniela-venuta
+Minh-Tam Do           | @mint47
+Luc Nguyen            | @1999lucnguyen  
+Karin Kazarian        | @Karin-kazarian  
 
 ### Description
-This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+This is a social media project based on the main features of instagram. This project is an introduction that we created to help us learn  the basics of industrial software development process.
```

Comment:
Two things:
1. Instagram needs to be capitalized
2. Can we create an introduction to help us learn something? Think you meant either "This project aims for us to learn the basics of the industrial software development process." or "This project was an introduction to the basics of the industrial development process."